app/views/prison/visits/_prisoner_details.html.erb

```diff
@@ -50,5 +50,11 @@
   <div class="column-one-quarter">
     <span class="text-secondary"><%= t('.prison_name') %></span>
     <div class="bold-small"><%= @visit.prison_name %></div>
+    <% case @visit.location_confirmed? %>
```

Comment:
I think `location_confirmed` will need to be delegated in the visit decorator for this to work